Difi.Felles.Utility/CertificateValidator.cs

```diff
@@ -14,8 +14,48 @@ public static bool IsValidCertificate(X509Certificate2 certificate, string certi
             return ValidateCertificate(certificate, certificateOrganizationNumber).Type == CertificateValidationType.Valid;
         }
 
+        /// <summary>
+        ///     Validates the certificate and chain. Validates that certificate is
+        ///     <list type="bullet">
+        ///         <item> Not null </item>
+        ///         <item> Issued to organization number </item>
+        ///         <item> Is activated </item>
+        ///         <item> Not Expired </item>
+        ///         <item> Has a valid chain. A valid chain is one built with the allowed chain certificates.</item>
```

Comment:
Nitpick, men dok-en her kunne tålt en liten omskriving :) Når «is» er en del av første setning, er det to punkter som leser litt rart:

- «Validates that certficiate is Is activated»
- «Validates that certificate is Has a valid chain.»

Hva med å endre første setning til bare «Validates that certificate», og så endre punktene til å være «… is not null», «… is issued to organization number» etc.?

Kan også vurdere å skrive om siste punkt til å bli én setning, slik som de andre punktene.
Fjern punktum til slutt for at det skal være likt for alle?

Dette gjelder forøvrig alle metodene i denne klassen :)